ubuntu@mtk-genio:~/Desktop/test/benchmark_dla$ sudo python3 test.py --auto
Traceback (most recent call last):
  File "/home/ubuntu/Desktop/test/benchmark_dla/test.py", line 194, in <module>
    benchmark_tflite_model(file, args.target, args.count, compile_options, cache_dla, args.profile)
  File "/home/ubuntu/Desktop/test/benchmark_dla/test.py", line 99, in benchmark_tflite_model
    gen_input_bin(modeil_io, input_bin)
NameError: name 'gen_input_bin' is not defined. Did you mean: 'input_bin'?

------------------------------------

ubuntu@mtk-genio:~/Desktop/test/benchmark_dla$ python3 quantize.py 
Traceback (most recent call last):
  File "/home/ubuntu/Desktop/test/benchmark_dla/quantize.py", line 7, in <module>
    converter = tf.lite.TFLiteConverter.from_saved_model(saved_model_dir)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/tensorflow/lite/python/lite.py", line 2073, in from_saved_model
    saved_model = _load(saved_model_dir, tags)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/tensorflow/python/saved_model/load.py", line 900, in load
    result = load_partial(export_dir, None, tags, options)["root"]
  File "/home/ubuntu/.local/lib/python3.10/site-packages/tensorflow/python/saved_model/load.py", line 1005, in load_partial
    loader_impl.parse_saved_model_with_debug_info(export_dir))
  File "/home/ubuntu/.local/lib/python3.10/site-packages/tensorflow/python/saved_model/loader_impl.py", line 59, in parse_saved_model_with_debug_info
    saved_model = parse_saved_model(export_dir)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/tensorflow/python/saved_model/loader_impl.py", line 119, in parse_saved_model
    raise IOError(
OSError: SavedModel file does not exist at: yolov5x.tflite/{saved_model.pbtxt|saved_model.pb}

-----------------------------------

ubuntu@mtk-genio:~/Desktop/test/benchmark_dla$ python3 quantize.py 
Traceback (most recent call last):
  File "/home/ubuntu/Desktop/test/benchmark_dla/quantize.py", line 4, in <module>
    converter = tf.lite.TFLiteConverter.from_tflite_model("yolov5x.tflite")
AttributeError: type object 'TFLiteConverterV2' has no attribute 'from_tflite_model'. Did you mean: 'from_keras_model'?

-----------------------------------------

ubuntu@mtk-genio:~/Desktop/test/benchmark_dla$ onnx-tf convert -i yolov5x.onnx -o yolov5x_tf_saved_model
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/onnx-tf", line 5, in <module>
    from onnx_tf.cli import main
  File "/home/ubuntu/.local/lib/python3.10/site-packages/onnx_tf/__init__.py", line 1, in <module>
    from . import backend
  File "/home/ubuntu/.local/lib/python3.10/site-packages/onnx_tf/backend.py", line 29, in <module>
    from onnx_tf.common.handler_helper import get_all_backend_handlers
  File "/home/ubuntu/.local/lib/python3.10/site-packages/onnx_tf/common/handler_helper.py", line 3, in <module>
    from onnx_tf.handlers.backend import *  # noqa
  File "/home/ubuntu/.local/lib/python3.10/site-packages/onnx_tf/handlers/backend/hardmax.py", line 3, in <module>
    import tensorflow_addons as tfa
ModuleNotFoundError: No module named 'tensorflow_addons'


---------------------------------------------

ubuntu@mtk-genio:~/Desktop/test/benchmark_dla$ pip install tensorflow-addons
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement tensorflow-addons (from versions: none)
ERROR: No matching distribution found for tensorflow-addons

----------------------------------------

# Install dependencies if not already installed
sudo apt-get install build-essential cmake

# Clone the tensorflow-addons repository
git clone https://github.com/tensorflow/addons.git

# Navigate into the tensorflow-addons directory
cd addons

# Install using pip
pip install .


OPPURE 


pip install tf2onnx

python -m tf2onnx.convert --opset 13 --input yolov5x.onnx --output yolov5x_tf_saved_model --saved-model

